quantum/backlight/backlight_timer.c

```diff
@@ -0,0 +1,166 @@
+#include "quantum.h"
+#include "backlight.h"
+#include "debug.h"
```

Comment:
```suggestion
#include "debug.h"
#include "backlight_driver_common.h"
```

Reply:
... Clearly I should have tested after my rebase